sources/packages/web/src/views/institution/locations/offerings/OfferingRequestChange.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Program detail"
+        :routeLocation="getRouteLocation()"
```

Comment:
Between a method and a computed please chose a computed when the result is based on props or other reactive properties.